hax/hax/ha/handler/node.py

```diff
@@ -27,6 +28,14 @@
 
 LOG = logging.getLogger('hax')
 
+__all__ = ['NodeEventHandler']
+
+statusMap: Dict[str, ServiceHealth] = {
+    'online': ServiceHealth.OK,
+    'offline': ServiceHealth.FAILED,
```

Comment:
offline maps to transient in motr, so let's send ServiceHealth.OFFLINE for `offline`.

Reply:
Addressed. Thanks!